lib/lettering/categories.py

```diff
@@ -7,7 +7,7 @@
 
 
 class FontCategory:
-    def __init__(self, cat_id=None, name=None):
+    def __init__(self, cat_id=None, name=None) -> None:
```

Comment:
Interesting... is it necessary to declare the return type of `__init__()`?  As far as I understood it's not supposed to return anything?

Reply:
Got it, thanks for the explanation!